web/client/components/mapcontrols/search/SearchResultList.jsx

```diff
@@ -7,15 +6,19 @@ const PropTypes = require('prop-types');
  * LICENSE file in the root directory of this source tree.
  */
 
-var React = require('react');
-var SearchResult = require('./SearchResult');
+const React = require('react');
```

Comment:
Since you are refactoring, why not moving to import / export instead of using require?
Just a suggestion